lib/checkVersions.js

```diff
@@ -12,9 +12,9 @@ const semver = require('semver');
 
 // DEFINE minimum versions here:
 /** The minimum required Node.JS version - should be the current LTS */
-const MIN_NODE_VERSION = '8.12';
+const MIN_NODE_VERSION = '12.13.0';
```

Comment:
hm ... 12 or better 10 because minimum in fact is 10 (and 10 is still LTS) @AlCalzone What do you think?

Reply:
Yes, 10 is still (maintenance) LTS, so I think this should be 10.21.0

Reply:
okay, so I will update that to 10.21.0 as it is the latest 10.x LTS version for now. The recommended versions are correct, right?